<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFormClass</class>
 <widget class="QMainWindow" name="MainFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>619</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>IisuOscBridge</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resource Files/MainFormResources.qrc">
    <normaloff>:/SK_logo_256x256.ico</normaloff>:/SK_logo_256x256.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/*----------------------------------------------------------------------------*/ 
/*----------------------------------------------------------------------------*/
QWidget
{
background: rgb(64, 64, 64);
color: rgb(220, 220, 220);
font: 8pt &quot;MS Shell Dlg 2&quot;;
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QMenuBar
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(120, 120, 120), stop:1 rgb(100, 100, 100));
}

QMenuBar::item
{
background: transparent;
color: rgb(220, 220, 220);
}

QMenu
{
background-color: rgb(60, 60, 60);
color: rgb(220, 220, 220);
border: 0px;
padding: 5px;
}

QMenu::item
{
background-color: transparent;
border: 1px solid transparent;
padding: 2px 25px 2px 20px;
}

QMenu::item:checked
{
background-color: rgb(120, 120, 120);
}

QMenu::item:selected
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(180, 180, 180), stop:1 rgb(130, 130, 130));
color: rgb(220, 220, 220);
}

QMenu::item:disabled
{
color: rgb(120, 120, 120);
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QDockWidget
{
background-color: rgb(90, 90, 90);
border-top: 1px solid orange;
color:rgb(220, 220, 220);
}

QDockWidget::title
{
background-color: rgb(90, 90, 90);
border: 0px;
padding: 0.4em;
}

QDockWidget::title:disabled
{
color:rgb(120, 120, 120);
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QTabBar::tab
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(90, 90, 90), stop:1 rgb(120, 120, 120));
border: 1px solid black;
color:rgb(220, 220, 220);
border-top-color: #444;
border-bottom-left-radius: 4px;
border-bottom-right-radius: 4px;
min-width: 8px;
padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(150, 150, 150), stop:1 rgb(180, 180, 180));
color:rgb(255,255,255);
}

QTabBar::tab:selected
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(120, 120, 120), stop:1 rgb(150, 150, 150));
}

QTabBar::tab:!selected
{
margin-top: 2px; /* make non-selected tabs look smaller */
}

QTabBar::tab:disabled
{
color:rgb(120, 120, 120);
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QPushButton
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(90, 90, 90), stop:1 rgb(60, 60, 60));
border: 1px solid black;
color:rgb(220, 220, 220);
padding: 1ex 2ex;
}

QPushButton:pressed
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(120, 120, 120), stop:1 rgb(90, 90, 90));
}

QPushButton:checked
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(150, 150, 150), stop:1 rgb(120, 120, 120));
}

QPushButton[text=&quot;...&quot;]
{
min-width: 1em;
}

QPushButton::hover
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(180, 180, 180), stop:1 rgb(130, 130, 130));
color:rgb(255,255,255);
}

QPushButton:disabled
{
color:rgb(120, 120, 120);
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QComboBox
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(90, 90, 90), stop:1 rgb(60, 60, 60));
border: 1px solid black;
color:rgb(220, 220, 220);
padding: 1ex 2ex;
}

QComboBox::drop-down
{
background-color: rgb(110, 110, 110);
border: 1px solid rgb(65, 65, 65);
}

QComboBox:on
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(180, 180, 180), stop:1 rgb(150, 150, 150));
}

QComboxBox::down-arrow
{
image: url(:/iisu_Toolbox/png/16x16/actions/go-down-8.png);
}

QComboxBox::down-arrow:on
{
top: 1px;
left: 1px;
}

QComboBox QAbstractItemView
{
selection-background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(180, 180, 180), stop:1 rgb(150, 150, 150));
}

QComboBox:disabled
{
color:rgb(120, 120, 120);
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QRadioButton
{
color: rgb(220, 220, 220);
}

QRadioButton:disabled
{
color:rgb(120, 120, 120);
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QAbstractItemView
{
background: rgb(96, 96, 96);
}

QAbstractItemView QCheckBox
{
margin-left: 5px;
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QTreeView
{
background-color: rgb(100, 100, 100);
alternate-background-color: rgb(90, 90, 90);
border: 0px solid black;
}

QTreeView:active
{
selection-background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(80, 80, 80), stop:1 rgb(60, 60, 60));
}

QTreeView:!active
{
selection-background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(80, 80, 80), stop:1 rgb(70, 70, 70));
}

QTreeView::item
{
color: rgb(220, 220, 220);
border: 0px solid green;
}

QTreeView::item:selected:!active
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(70, 70, 70), stop:1 rgb(50, 50, 50));
}

QTreeView::item:selected:active
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(80, 80, 80), stop:1 rgb(60, 60, 60));
}

QTreeView::item:hover
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(80, 80, 80), stop:1 rgb(60, 60, 60));
}

QTreeView::item:disabled
{
color:rgb(120, 120, 120);
}

QTreeView::branch:has-children:!has-siblings:closed, QTreeView::branch:closed:has-children:has-siblings
{
border-image: none;
image: url(:/iisu_Toolbox/png/stylesheet-branch-closed.png);
}

QTreeView::branch:open:has-children:!has-siblings, QTreeView::branch:open:has-children:has-siblings
{
border-image: none;
image: url(:/iisu_Toolbox/png/stylesheet-branch-open.png);
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QHeaderView::section
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(90, 90, 90), stop:1 rgb(60, 60, 60));
border: 1px solid black;
color:rgb(220, 220, 220);
padding: 1ex 2ex;
}

QHeaderView::section:disabled
{
color:rgb(120, 120, 120);
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QScrollBar
{
background-color: rgb(55, 55, 55);
}

QScrollBar::add-page, QScrollBar::sub-page
{
background: none;
}

QScrollBar::horizontal
{
border:0px solid rgb(90, 90, 90);
border-left: 0px solid red;
border-right: 0px solid red;

margin: 0px 16px 0px 16px;
}

QScrollBar::vertical
{
border: 0px solid rgb(90, 90, 90);
border-top: 0px solid red;
border-bottom: 0px solid red;

margin: 16px 0px 16px 0px;
}

QScrollBar::handle
{
background-color: rgb(110, 110, 110);
border: 1px solid rgb(65, 65, 65);
}

QScrollBar::handle:horizontal
{
min-width: 20px;
}

QScrollBar::handle:vertical
{
min-height: 20px;
}

QScrollBar::handle:hover
{
background-color: rgb(130, 130, 130);
}

QScrollBar::handle:pressed
{
background-color: rgb(180, 180, 180);
}

QScrollBar::add-line:horizontal
{
background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 rgb(60, 60, 60), stop:1 rgb(90, 90, 90));
border: 1px solid rgb(90, 90, 90);
border-left:0px;
width: 15px;
subcontrol-position: right;
subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal
{
background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 rgb(90, 90, 90), stop:1 rgb(60, 60, 60));
border: 1px solid rgb(90, 90, 90);
border-right:0px;

width: 15px;
subcontrol-position: left;
subcontrol-origin: margin;
}

QScrollBar::add-line:horizontal:hover
{
background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 rgb(100, 100, 100), stop:1 rgb(150, 150, 150));
}

QScrollBar::sub-line:horizontal:hover
{
background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 rgb(150, 150, 150), stop:1 rgb(100, 100, 100));
}

QScrollBar::add-line:horizontal:pressed
{
background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 rgb(130, 130, 130), stop:1 rgb(180, 180, 180));
}

QScrollBar::sub-line:horizontal:pressed
{
background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 rgb(180, 180, 180), stop:1 rgb(130, 130, 130));
}

QScrollBar::add-line:vertical
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(60, 60, 60), stop:1 rgb(90, 90, 90));
border: 1px solid rgb(90, 90, 90);
border-top:0px;
height: 15px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(90, 90, 90), stop:1 rgb(60, 60, 60));
border: 1px solid rgb(90, 90, 90);
border-bottom:0px;
height: 15px;
subcontrol-position: top;
subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(100, 100, 100), stop:1 rgb(150, 150, 150));
}

QScrollBar::sub-line:vertical:hover
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(150, 150, 150), stop:1 rgb(100, 100, 100));
}

QScrollBar::add-line:vertical:pressed
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(130, 130, 130), stop:1 rgb(180, 180, 180));
}

QScrollBar::sub-line:vertical:pressed
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(180, 180, 180), stop:1 rgb(130, 130, 130));
}

QScrollBar::left-arrow:horizontal
{
background-position: center;
}

QScrollBar::right-arrow
{
background-position: center;
}

QScrollBar::up-arrow:vertical
{
background-position: center;
image: url(:/iisu_Toolbox/png/16x16/actions/go-up-8.png);
}

QScrollBar::down-arrow:vertical
{
background-position: center;
image: url(:/iisu_Toolbox/png/16x16/actions/go-down-8.png);
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QGroupBox
{
border: 1px solid black;
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QSlider::groove:horizontal
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(180, 180, 180), stop:1 rgb(140, 140, 140));
border: 0px solid rgb(60, 60, 60);
height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
}

QSlider::handle:horizontal
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(180, 180, 180), stop:1 rgb(140, 140, 140));
border: 1px solid rgb(60, 60, 60);
width: 8px;
margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
border-radius: 2px;
}

QSlider::handle:horizontal:hover
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(220, 220, 220), stop:1 rgb(180, 180, 180));
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QSpinBox
{
border: 1px solid black;
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QLCDNumber
{
border: 1px solid black;
color: rgb(220, 220, 220);
}

QLCDNumber:disabled
{
color:rgb(120, 120, 120);
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QGraphicsView
{
border: 0px;
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
QToolTip
{
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgb(180, 180, 180), stop:1 rgb(150, 150, 150));
}

/*----------------------------------------------------------------------------*/
/*----------------------------------------------------------------------------*/
#SK__Toolbox__EventsView, #SK__Toolbox__EventsView &gt; #eventsView, #SK__Toolbox__EventsView &gt; #eventsLabel, #SK__Toolbox__EventsView &gt; #toggle,
#SK__Toolbox__EventsFilter, #SK__Toolbox__EventsFilter &gt; QWidget
{
background: rgb(40, 40, 40);
color: rgb(220, 220, 220);
}

#SK__Toolbox__EventsView &gt; #eventsLabel
{
font: 16pt &quot;MS Shell Dlg 2&quot;;
}

#SK__Toolbox__EventsView &gt; #eventsView
{
border: 0px;
}

#SK__Toolbox__EventsView &gt; #toggle
{
padding: 1ex 2ex;
border: 0px;
}

#SK__Toolbox__EventsFilter, #SK__Toolbox__EventsFilter &gt; QWidget
{
font: 12pt &quot;MS Shell Dlg 2&quot;;
}

#info
{
background: rgb(220, 220, 220);
color: rgb(40, 40, 40);
padding: 1ex 2ex;
font:20pt &quot;MS Shell Dlg 2&quot;;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_startStopToggleButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../Resource Files/MainFormResources.qrc">
          <normaloff>:/Play_icon_32x32.png</normaloff>
          <normalon>:/Stop_icon_32x32.png</normalon>:/Play_icon_32x32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>IID :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_iidFilePathEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="m_iidFilePathButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Port :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_portEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>IP :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_ipAddressEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="m_tabs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="m_mapsTab">
       <attribute name="title">
        <string>Maps</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QTreeView" name="m_pathsView"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="m_addMapButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_insertMapButton">
              <property name="text">
               <string>Insert</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_deleteMapButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="m_foldAndNameJointsCheckBox">
            <property name="text">
             <string>Fold and Name Joints</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>OSC Packet Size :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_oscPacketSizeLineEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Bytes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="m_logTab">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPlainTextEdit" name="m_logTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../Resource Files/MainFormResources.qrc"/>
 </resources>
 <connections/>
</ui>
